src/brevitas/graph/equalize.py

```diff
@@ -222,27 +352,34 @@ def _extract_regions(graph_model: GraphModule):
         if node.op == 'call_module':
             module = get_module(graph_model, node.target)
             if isinstance(module, _supported_layers):
-                srcs, sinks = {node.target}, set()
+                srcs = {node.target: 0}
+                sinks = dict()
                 walk_region(graph_model, node, set(), srcs, sinks, walk_forward=True)
                 if sinks:
                     # each region should appear only once, so to make it hashable
                     # we convert srcs and sinks to ordered lists first, and then to tuples
-                    regions.add((tuple(sorted(srcs)), tuple(sorted(sinks))))
-    # for clarity, sort by the of the first source
+                    srcs_ordered = [0] * len(srcs.items())
+                    for v, k in srcs.items():
+                        srcs_ordered[k] = v
+                    regions.add((tuple(srcs_ordered), tuple(sorted(sinks.keys()))))
     regions = sorted(regions, key=lambda region: region[0][0])
     return regions
 
 
+
 class EqualizeGraph(GraphTransform):
 
-    def __init__(self, iterations) -> None:
+    def __init__(self, iterations, threshold=None, merge_bias=True, bias_shrinkage='vaiq') -> None:
```

Comment:
We need to provide a good default threshold and number of iterations. Something like 0.95 and 10?